### PR TITLE
Add article tagging prototype extension

### PR DESCRIPTION
**Pull request by CharlieHawker**
This forms the first prototype UI Extension written for Contentful as a proof of concept for the Guide to Parliament Beta project.

Rewritten after feedback to use a Makefile and remove dependency on jQuery.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+.DS_Store
+public/dist
```
Comment (jakemulley) [inline .gitignore:3]: You probably want to add `article-tagging-prototype/dist` here too, as you're currently pushing compiled files

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Common variables
+NODE_MODULES=./node_modules
+PUBLIC_FOLDER=./public
+DIST_FOLDER=$(PUBLIC_FOLDER)/dist
```
Comment (jakemulley) [inline Makefile:4]: `article-tagging-prototype/dist`, again

**Review by jakemulley — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (CharlieHawker) [reply .gitignore:3]: @jakemulley that whole `article-tagging-prototype` folder that shouldn't have been in the repo; apologies. It dated from before the repo was to host multiple UI extensions. Have discarded and re-pushed.

File: Makefile (hunk 2 above)
Comment (CharlieHawker) [reply Makefile:4]: Again `article-tagging-prototype` folder should not have been in the repo at all; removed.

File: public/tagging/extension.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "tagging",
```
Comment (naseberry) [inline public/tagging/extension.json:2]: It would be ideal to prefix this with 'UK_Parliament'

File: src/data/terms.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+const termsPromise = (() => {
+  return new Promise((resolve, reject) => {
+    resolve([
```
Comment (naseberry) [inline src/data/terms.js:5]: As this is a prototype, are there any plans to improve these key:value?

**Pull request closed without merging**